js/full-picture-render.js

```diff
@@ -1,9 +1,12 @@
 import {isEscapeKeydown} from './functions.js';
 
+const NUMBER_OF_VISIBLE_COMMENTS = 5;
+
```

Comment:
добавь переменную, в которой будем считать количество отрисованных комментариев

```suggestion
let visibleCommentsCounter = NUMBER_OF_VISIBLE_COMMENTS;
```